src/main/java/levy/fallingsand/Main.java

```diff
@@ -5,6 +5,9 @@
     public class Main {
 
         public static void main(String[] args) {
+            //new SandFrame().setVisible(true);
+            SandFrame frame = new SandFrame();
+            frame.setVisible(true);
```

Comment:
Like I said. 
Do not put this in your command line program.

Reply:
it simply said create a gui so i did like we did for projectile. I am confused

Reply:
I didn't say combine it with your command line program. Create a new class with a main